packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -177,15 +174,15 @@ void main() {
     );
     await tester.tap(find.byType(EditableText));
     await tester.showKeyboard(find.byType(EditableText));
-    controller.text = 'test';
+   controller.text = 'test';
```

Comment:
nit: something got messed up with formatting here and on many other lines that begin with "controller".

Reply:
Darn search/replace! Fixed. I searched through to check all of them too, just in case.